.gitignore

```diff
@@ -161,7 +161,7 @@ docs/source/apidocs
 *.nii.gz
 .DS_Store
 *.DS_Store
-bin
+#bin
```

Comment:
lets restore this.. any files u add you should use git add -f option

Reply:
Ok, done.